backend/src/scripts/backup-mongodb.ts

```diff
@@ -0,0 +1,91 @@
+import * as Minio from 'minio';
+import * as child from 'child_process';
+import * as Fs from "fs";
+import { config } from "../config";
+
+const ensureBucketExists = async (client : Minio.Client) => {
+    const exists = await client.bucketExists(config.S3_MONGO_BACKUP_BUCKET)
+    if (!exists) {
+        const res = await client.makeBucket(config.S3_MONGO_BACKUP_BUCKET)
+        console.log(`Bucket ${config.S3_MONGO_BACKUP_BUCKET} created`)
+    } 
+}
+
+const error = async (client : Minio.Client, toDelete : string[]) => {
+    client.removeObjects(config.S3_MONGO_BACKUP_BUCKET, toDelete, (e) => {
+        if (e) {
+            console.error("Unable to delete incorrect objects. Please do so manually.")
+        }
+    });
+}
+
+(async () => {
+
+    const client = new Minio.Client({
```

Comment:
is this compatible with any S3 service? we are only using minio for local testing